### PR TITLE
Bootstrap integration

### PR DESCRIPTION
**Pull request by tboerger**
This pull request handles the bootstrap integration for the barclamp. **It depends on https://github.com/crowbar/barclamp-crowbar/pull/790**

### CODE REVIEW

**Comment by dpaterson** [comment]
Don't think RAID and BIOS can be touched,  see Arkady's state of roxy email on the crowbar list:
"b. Barclamp_dell_raid and barclamp_dell_bios – any changes to these MUST be reviewed and approved by Dell. First, the current repos for them are shared for multiple releases: roxy, hadoop-2.4 and hydrogen. Thus, any change impacts all three. Second in order to test it actual HW testing is required and only Dell folks know how to test changes in these barclamps. It is best not to touch these by anybody but Dell folks."

**Comment by arkadykanevsky** [comment]
-1 this change will break other branches that use this repo.

**Comment by tboerger** [comment]
But it only touches the roxy branch to get the bootstrap branding here too?

File: crowbar_framework/app/views/barclamp/dell_bios/_edit_attributes.html.haml [hunk 1]
```diff
@@ -1,23 +1,8 @@
...
-      %label{ :for => :debug }= t('.debug')
-      = select_tag :debug, options_for_select([['true','true'], ['false', 'false']], @proposal.raw_data['attributes'][@proposal.barclamp]["debug"].to_s), :onchange => "update_value('debug', 'debug', 'boolean')"
-
-
+= attributes_for @proposal do
+  .panel-sub
+    = header @dep_raw, true
+
+  .panel-body
+    = boolean_field :bios_setup_enable
```
Comment (vuntz) [inline crowbar_framework/app/views/barclamp/dell_bios/_edit_attributes.html.haml:6]: bios_update_enable seems to be dropped? Is this wanted?

Comment (tboerger) [inline crowbar_framework/app/views/barclamp/dell_bios/_edit_attributes.html.haml:6]: Fixed!

**Pull request merged**